xbmc/addons/AddonManager.h

```diff
@@ -486,6 +486,14 @@ namespace ADDON
     bool GetCompatibleVersions(const std::string& addonId,
                                std::vector<std::shared_ptr<IAddon>>& compatibleVersions) const;
 
+    /*!
+     * \brief Return number of available updates collected by GetAvailableUpdates()
```

Comment:
i would never describe any implementation details in interface documentation, only the functionality.